packages/ra-core/src/util/Ready.tsx

```diff
@@ -65,37 +65,41 @@ const Button = ({ img, label, href }) => {
+                    alt="react-admin logo"
+                />
+                <h1>Welcome to React-admin</h1>
+                <div>
+                    Your application is properly configured.
```

Comment:
We don't translate these texts?

Reply:
Because at that stage it's probable that the admin hasn't any i18n. The default Ready component is meant for development purposes only.